packages/pf-codemods/README.md

```diff
@@ -480,9 +480,9 @@ Out:
 <SimpleList isActive />
 ```
 
-### spinner-svg-default [(#8183)](https://github.com/patternfly/patternfly-react/pull/8183)
+### spinner-remove-issvg [(#8616)](https://github.com/patternfly/patternfly-react/pull/8616)
 
-We've updated the Spinner to default to an svg, so the `isSVG` property is no longer required.
+We've updated the Spinner to only be an svg and have removed the isSVG prop.
```

Comment:
I thought "only be" would make it more clear that it cannot be anything else. When I hear "default", it would make me think there is still a way to change it.

Reply:
Ahh that certainly makes sense and you make a great point. In that case, what do you think of something like:
```suggestion
We've updated the Spinner to exclusively use an SVG, and have removed the isSVG prop.
```